public/index.html

```diff
@@ -88,48 +85,25 @@ <h3>Share With Others</h3>
         </ul>
         <footer class="major">
           <ul class="actions special">
-            <li><a href="#cta" class="button" id='get-started'>Get Started</a></li>
+            <li><a href="#get-started" class="button" id='get-started'>Get Started</a></li>

```

Comment:
`id` needs to be changed from `get-started`. `hrefs` are now scrolling to this button instead of the `iframe`